test/src/test_pylint.py

```diff
@@ -57,6 +57,19 @@ def test_pylint(source_files):
         pytest.fail("pylint issues detected")
 
 
+@pytest.mark.skipif(not test.TestBase.have_isort(), reason="isort not available")
+def test_isort(source_files):
+    #
+    # Run `isort` on all python sources. We simply use `find` to locate
+    # all `*.py` files, and then manually select the reverse-domain named
+    # modules we have.
+    #
```

Comment:
Lets maybe stop copying this comment everywhere?

Reply:
Agree.